jupyterhub/schema.yaml

```diff
@@ -474,9 +476,7 @@ properties:
               zhy270a
               ```
           pullPolicy:
```

Comment:
I believe a null pull policy is valid, and will use kubernetes' default pull policy (i.e. Always if tag == latest, IfNotPresent otherwise).

Reply:
Hmmm, the valid options for k8s are _unset_, `IfNotPresent`, `Always`, `Never`.

The validation logic for this Helm chart config will of course always accept the valid non-empty string values, the question is with regards to the _unset_ part. It must be able to accept the unset part, but should it accept a blank string to imply unset, a null value to imply unset, or both?

I think the right call is to accept both, so I'll update the type field below to include `"null"` so it would accept both a blank string and `"null"`, sounds okay?

Reply:
Resolved by f3938781